sections/featured-product.liquid

```diff
@@ -325,8 +372,15 @@
                             value="{{ variant.id }}"
                           >
                             {{ variant.title }}
-                            {%- if variant.available == false %} - {{ 'products.product.sold_out' | t }}{% endif %}
                             - {{ variant.price | money | strip_html }}
+                            {%- if variant.available == false %} - {{ 'products.product.sold_out' | t }}{% endif %}
+                            {% if variant.available and variant.inventory_management == 'shopify' -%}
+                              -
+                              {{
+                                'products.product.inventory_in_stock_show_count'
+                                | t: quantity: variant.inventory_quantity
```

Comment:
This should be matching the block setting. Or since we can't make it match since it's based on a setting from a different block we should default to not showing the inventory quantity. Just `in stock` if above `0`.

Reply:
In fact you can probably do a else rather than a second if statement: 
```html
{%- if variant.available == false %} - {{ 'products.product.sold_out' | t }}{% elsif variant.available and variant.inventory_management == 'shopify' -%}
                              -
                              {{
                                'products.product.inventory_in_stock'
                                | t

Reply:
Thanks for catching this. I added a full logic to cover all three cases when "Show count" on and off, and when it's set to continue selling even if it's sold out. It may be too much code and unnecessary for no JS case though. What do you think @ludoboludo . Should I keep it or make it simple and show just a status In Stock/Low Stock?